akka-actor-typed/src/main/scala/akka/actor/typed/SpawnProtocol.scala

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.typed.scaladsl.Behaviors
+
+object SpawnProtocol {
+
+  object Spawn {
+    /**
+     * Special factory to make using Spawn with ask easier
+     */
+    def apply[T](behavior: Behavior[T], name: String, props: Props): ActorRef[ActorRef[T]] ⇒ Spawn[T] =
+      replyTo ⇒ new Spawn(behavior, name, props, replyTo)
+  }
+
+  /**
+   * Spawn a child actor with the given `behavior` and send back the `ActorRef` of that child to the given
+   * `replyTo` destination.
+   *
+   * If `name` is an empty string an anonymous actor (with automatically generated name) will be created.
+   */
+  final case class Spawn[T](behavior: Behavior[T], name: String, props: Props, replyTo: ActorRef[ActorRef[T]])
+    extends SpawnProtocol {
+
+  }
```

Comment:
Should it be possible to just fire and forget start as well, or is that never interesting?

Reply:
I don't think that should be very common and then this new ticket may come in handy https://github.com/akka/akka/issues/25306

Reply:
Im fine with the shape of the case class like this 

nit: could remove {}